### ISSUE TITLE
Expression form is not shown

### ISSUE THREAD

**Issue by kidiboy** [grafana/grafana#63779]
**What happened**: 
After clicking "+ Expression" I don't see the expression configuration form:

<img width="1033" alt="empty_expression" src="https://user-images.githubusercontent.com/55297357/221432262-c5178986-5fbc-4096-8998-0b86ed20338a.png">

**What you expected to happen**:

I expected to see some form where I could configure the expression

**How to reproduce it (as minimally and precisely as possible)**:

1. `docker-compose.yml`:
```yaml
  grafana:
    image: grafana/grafana:9.3.6
    ports:
      - 3000:3000
```
2. Add some Prometheus datasource
3. Create a dashboard
4. Add a panel, add 2 arbitrary queries
5. Click "+ Expression"
6. Expression form is not shown

**Environment**:
- Grafana version: v9.3.6 (978237e7cb)
- Data source type & version: Prometheus (VictoriaMetrics v1.86.2-cluster)
- OS Grafana is installed on: Linux (docker)
- User OS & Browser: MacOS 12.6.3, Google Chrome Version 110.0.5481.177 (Official Build) (arm64)
- Grafana plugins: no
- Others: –

**Comment by davidgourde** [comment]
I'm having the same problem on my end

**Comment by kikkof** [comment]
The same problem was reported here https://github.com/grafana/grafana/issues/62259

**Comment by zuchka** [comment]
[see comment in this thread](https://github.com/grafana/grafana/issues/63227#issuecomment-1439901418)
> will be fixed in 9.3.7
